vmm/src/config.rs

```diff
@@ -1603,6 +1625,105 @@ impl FsConfig {
     }
 }
 
+#[cfg(feature = "fw_cfg")]
+impl FwCfgConfig {
+    pub const SYNTAX: &'static str = "Boot params to pass to FW CFG device \
+    \"disable<optional; enabled if not set>,e820=on|off,kernel=on|off,cmdline=on|off, \
```

Comment:
I don't see why we need a `disable` option - The `fw_cfg` device is created when and only when `--fw-cfg-config` option is provided.